test/test_siteset.jl

```diff
@@ -7,9 +7,22 @@ using ITensors,
 
   @testset "Star in operator strings" begin
     sites = spinHalfSites(N)
+    @test_throws ArgumentError op(sites, "Sp")
```

Comment:
Hi Katie, not sure what this is intended to test. There isn't a method `op(::SiteSet,::String)`; it requires an integer argument at the end too. So right now it's throwing a MethodError and failing the test.

Reply:
Ah derp, sorry about that! Early AM testing 😰